packages/runtime/src/next/components/page.tsx

```diff
@@ -11,7 +11,13 @@ export type PageProps = {
   snapshot: MakeswiftPageSnapshot
 }
 
-export const Page = memo(({ snapshot }: PageProps) => {
+export const Page = memo(({ snapshot, ...props }: PageProps) => {
+  if ('runtime' in props) {
+    const message =
+      'The `runtime` prop is no longer supported in the @makeswift/runtime `Page` component as of `0.15.0`.'
+    console.error(message)
+    throw new Error(message)
```

Comment:
Do we need both? Can we just throw the error and have it logged?

Reply:
No we don't need both. Updated!

I was looking at the Nextjs repo for how they did similar exceptions and they did both, I just c/p'd from there. But throwing an error logs the message anyway, so there is no reason to `console.error`

Reply:
Trueeeee. I think warning them makes more sense here. Now that you mention it!

Reply:
I originally suggested we throw an error because passing a prop like that is most likely due to a user that didn't migrate or hasn't migrated properly and a warning might be missed, specially in the other sea of warnings that we have. An error would make it a lot more obvious something is wrong. That being said, if the warning is prominent and hard to miss, it does feel better to do a warning than an error.

Reply:
Just make the warning prominent and I think we should be :gucci: